assets/src/edit-story/masks/index.js

```diff
@@ -64,6 +64,7 @@ export const MASKS = [
     type: MaskTypes.HEART,
     name: __('Heart', 'web-stories'),
     path: CLIP_PATHS[MaskTypes.HEART],
+    ratio: 0.92977124,
```

Comment:
Might be cleaner to have a width and height here (doesn't matter what scale) instead of a float

Reply:
I agree with that: let's get it closer to the viewBox with width and height.

Reply:
After discussion with Dima, I've decided to stay with a value between 0..1 for now, as that will make dealing with those viewboxes and subsequent transforms simpler.